arangod/Agency/AgentCallback.h

```diff
@@ -38,17 +38,14 @@ class AgentCallback final {
  public:
   AgentCallback();
 
-  AgentCallback(Agent*, std::string const&, index_t, size_t);
+  AgentCallback(Agent*, std::string, index_t, size_t);
```

Comment:
Also now 1 copy ctor + 1 move ctor instead of just 1 copy ctor.
I don't understand why, but ok